addons/project/views/project_project_views.xml

```diff
@@ -85,31 +85,34 @@
                         <page name="settings" string="Settings">
                             <group>
                                 <group>
-                                    <field name="privacy_visibility" widget="radio"/>
-                                    <span colspan="2" class="text-muted o_row ps-1" invisible="access_instruction_message == ''">
-                                        <i class="fa fa-lightbulb-o"/>&amp;nbsp;<field class="d-inline" name="access_instruction_message" nolabel="1"/>
-                                    </span>
-                                    <span colspan="2" class="text-muted o_row ps-1" invisible="privacy_visibility_warning == ''">
-                                        <i class="fa fa-warning"/>&amp;nbsp;<field class="d-inline" name="privacy_visibility_warning" nolabel="1"/>
-                                    </span>
-                                </group>
-                                <group>
-                                    <div name="alias_def" colspan="2" class="pb-2">
+                                    <div name="alias_def" colspan="2">
                                         <!-- Always display the whole alias in edit mode. It depends in read only -->
                                         <!-- Need to add alias_id in view for getting alias_domain_id by default -->
                                         <field name="alias_id" invisible="1"/>
-                                        <label for="alias_name" class="fw-bold o_form_label" string="Create tasks by sending an email to"/>
-                                        <field name="alias_email" class="oe_read_only d-inline" widget="email" readonly="1" invisible="not alias_name" />
-                                        <span class="oe_edit_only o_row">
+                                        <span class="o_wrap_label text-900">
```

Comment:
are you sure that this span is needed :thinking:

Reply:
ok thanks 🙂

Reply:
Actually, I did not see any difference if I remove that span. 🤔

Reply:
ah okay the text weight is different

Reply:
if we do this then we will have this issue


![image](https://github.com/user-attachments/assets/3c2ea1d7-ef55-4938-b468-d8c8c9dec964)